examples/AspNetCore/Program.cs

```diff
@@ -13,20 +13,20 @@
 var appBuilder = WebApplication.CreateBuilder(args);
 
 // Note: Switch between Zipkin/OTLP/Console by setting UseTracingExporter in appsettings.json.
-var tracingExporter = appBuilder.Configuration.GetValue("UseTracingExporter", defaultValue: "console").ToLowerInvariant();
+var tracingExporter = appBuilder.Configuration.GetValue("UseTracingExporter", defaultValue: "CONSOLE").ToUpperInvariant();
```

Comment:
Why does this need to be changed?

Reply:
Found the warning: [CA1308: Normalize strings to uppercase](https://learn.microsoft.com/en-us/dotnet/fundamentals/code-analysis/quality-rules/ca1308).